pages/linux/latexmk.md

```diff
@@ -2,18 +2,18 @@
 
 > Automated LaTeX document generation.
 
-- Compile filename.tex to pdf:
+- Compile a LaTeX file to PDF:
```

Comment:
What about `Render a LaTeX file as a PDF`?

Reply:
That could work, but I don't see what's wrong with the current wording. Is it the "compile" that you want to get rid of?

Reply:
It's mainly "to" which feels awkward actually, though compiling feels like compiling a program into it's executable :P

Reply:
Well, it certainly is converting a plaintext file into a binary format :) would you be OK with "into a PDF file", or do you still prefer the "render" variant? I'm fine with either.

Reply:
Yeah, "into a PDF file" would work just fine!